Scripts/assignment/freight_assignment.py

```diff
@@ -198,22 +201,50 @@ def _filter_border_points(self, border_data: dict) -> Dict[str, int]:
                     if border_data[pid]["id"] in zone_numbers}
         return port_ids
 
-    def _line_data_to_matrix(self, ship_mode: str,
-                             attribute: str,
+    def _get_marine_transit_lines(self, is_export: bool) -> dict:
+        """Fetch list of marine transit lines.
+
+        Parameters
+        ----------
+        is_export : bool
+            Fetch export specific transit lines. Else import specific
+        
+        Returns
+        -------
+        dict
+            marine mode : str
+                marine ship mode
+            transit lines : list
+                Emme Transit line objects
+        """
+        mode_transit_lines = defaultdict(list)
+        ship_modes = {mode for nesting in param.freight_marine_modes.values() 
+                      for mode in nesting}
+        for line in self.emme_scenario.get_network().transit_lines():
+            if line.mode.id not in ship_modes:
+                continue
+            
+            id_split = line.id.split("_")[0]
+            try:
+                int(id_split)
+                continue
+            except ValueError:
+                starts_in_fi = line.id[0:2] == "FI"
+                if ((starts_in_fi and is_export) or (not starts_in_fi and not is_export)):
+                    mode_transit_lines[line.mode.id].append(line)
```

Comment:
This logic seems a bit backwards. The thing inside the exception is what we normally want to perform, and the value being an `int` is what would break it, right? Could this be expressed more straightforward, for instance using `line.id[0:2].isalpha()` or similar?